app/components/big-text.js

```diff
@@ -8,28 +12,36 @@ export default Ember.Component.extend({
   expandIcon: 'info-circle',
   text: '',
   ellipsis: 'ellipsis-h',
-  lengths: Ember.computed.collect('length', 'slippage'),
-  totalLength: Ember.computed.sum('lengths'),
-  promptText: '',
-  showIcons: function(){
-    return this.get('displayText') !== this.get('cleanText');
-  }.property('displayText', 'text'),
-  cleanText: function(){
-    var text = this.get('text');
-    if(text === undefined || text == null){
-      return this.get('promptText')?this.get('promptText').toString():'';
+  lengths: collect('length', 'slippage'),
+  totalLength: sum('lengths'),
+  renderHtml: true,
+  showIcons: computed('displayText', 'text', 'renderHtml', function(){
+    if(this.get('renderHtml')){
+      return this.get('displayText').toString() !== this.get('text');
+    } else {
+      return this.get('displayText').toString() !== this.get('cleanText');
     }
+  }),
+  cleanText: computed('text', function(){
     //strip any possible HTML out of the text
-    return text.replace(/(<([^>]+)>)/ig,"");
```

Comment:
@stopfstedt jquery.text() is for working with elements.  We're working with a string passed from somewhere else like the DB.  This would lead to something like:
```return  Ember.$("<div/>").html(text).text();```  I'm not sure that isn't more confusing.  This isn't a security step, its purely to easy in the display.  Thoughts?